helm_deploy/prison-api/templates/deployment.yaml

```diff
@@ -48,15 +48,15 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: /elite2api/ping
+              path: /health/liveness
               port: {{ .Values.image.port }}
             periodSeconds: 30
             initialDelaySeconds: 90
             timeoutSeconds: 20
             failureThreshold: 10
           readinessProbe:
             httpGet:
-              path: /elite2api/ping
+              path: /health/readiness
```

Comment:
switch to readiness/liveness

Reply:
is this a in-built spring boot endpoint?